src/PostgreSqlTransportAcceptanceTests/ConfigureEndpointPostgreSqlTransport.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Npgsql;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting.Support;
+using NServiceBus.Transport;
+
+public class ConfigureEndpointPostgreSqlTransport : IConfigureEndpointTestExecution
+{
+    public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings, PublisherMetadata publisherMetadata)
+    {
+        transport = new TestingPostgreSqlTransport(async cancellationToken =>
+        {
+            var conn = PostgreSqlConnectionBuilder.Build();
+            await conn.OpenAsync(cancellationToken).ConfigureAwait(false);
+            return conn;
+        });
+
+        configuration.UseTransport(transport);
+
+        return Task.CompletedTask;
+    }
+
+    public Task Cleanup()
+    {
+        using (var conn = PostgreSqlConnectionBuilder.Build())
+        {
+            conn.Open();
+
+            //var queueAddresses = transport.ReceivingAddresses.ToList();
+            //foreach (var address in queueAddresses)
+            //{
+            //    TryDeleteTable(conn, address);
+            //    TryDeleteTable(conn, new QueueAddress(address.Table + ".Delayed", address.Schema, address.Catalog));
+            //}
```

Comment:
I think, this can be removed - as the container with PostgreSQL DB is setup on each run